### PR TITLE
FAQ - Difference between test and snapshot

### PR DESCRIPTION
**Pull request by irashid1**
https://linear.app/chromaui/issue/SUP-182/faq-%E2%94%80-whats-the-difference-between-test-and-snapshot

### CODE REVIEW

File: src/images/diff-between-test-and-snapshot.png
Comment (hannahpx) [inline src/images/diff-between-test-and-snapshot.png:1]: can we have the image added like the following so the main aspect of tests is shown
turbosnap-tests
 @irashid1

Comment (irashid1) [reply src/images/diff-between-test-and-snapshot.png:1]: Changed ✅

**Review by domyen — COMMENTED** [review]
Updated

**Review by winkerVSbecks — APPROVED** [review]

**Comment by domyen** [comment]
no worries, I was already here because I just submitted a PR myself so figured I could tidy up while I'm here 🧹

**Pull request merged**